documentation/wtpdf-from-latex.md

```diff
@@ -0,0 +1,251 @@
+---
+layout: "ngpdf"
+lang: "en"
+---
+
+<script>
+runlatex.editorlines=120;
+</script>
+
+# Generating Well Tagged PDF from LaTeX
+
+### An Initial Example
+
+To enable tagging in supported documents it is enough to add at the
+very beginning of the document, i.e., before `\documentclass`, a
+`\DocumentMetadata` declaration. It takes one argument, a key-value
+list.
+
+The first four keys in the example set important document metadata,
+like the language, the requested PDF version and the standards the
+document should comply with.
+
+Tagging is then enabled by loading various modules through the
+`testphase` key.
```

Comment:
The value `latest` loads all modules that we recommend so that it is not necessary to specify individual modules (the loaded modules are logged in the transcript file).